README.md

```diff
@@ -75,12 +75,11 @@ We recommend you use Toitware's [ESP-IDF fork](https://github.com/toitware/esp-i
 * Allocation-fixes for UART, etc.
 * LWIP fixes.
 
+This repository has been added as as submodule reference, so doing a recursive init & update will establish everything nedded:
```

Comment:
We should update the instructions for how to set IDF_PATH to indicate where this is found - and indicate that this is where you would choose a different `esp-idf` version if you want to tinker with it.

```
export IDF_PATH=`pwd`/third_party/esp-idf
```

Reply:
We could also let the Makefile give these instructions when it finds that `IDF_PATH` isn't set.

Reply:
Added as suggested, plus some copyediting.